### PR TITLE
added dark mode upgrade banner

### LINKED ISSUE #8163: [CAL-1441] dark banners for upgrade screens
CleanShot 2023-04-09 at 13 26 02@2x

<sub>[CAL-1441](https://linear.app/calcom/issue/CAL-1441/dark-banners-for-upgrade-screens)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
CleanShot%202023-04-10%20at%2021 50 37@2x

### CODE REVIEW

**Review by zomars — APPROVED** [review]

**Review by roae — APPROVED** [review]
LGTM!

File: packages/features/tips/UpgradeTip.tsx [hunk 1]
```diff
@@ -37,12 +33,14 @@ export function UpgradeTip({
 
   if (isParentLoading || isLoading) return <>{isParentLoading}</>;
 
+  const isDark = document.querySelector("html").classList.contains("dark");
```
Comment (PeerRich) [inline packages/features/tips/UpgradeTip.tsx:36]: this is failing lint, possibly null

Comment (roae) [reply packages/features/tips/UpgradeTip.tsx:36]: `const isDark = Boolean(document.querySelector("html").classList.contains("dark"));`

**Review by zomars — CHANGES_REQUESTED** [review]
Blocking to make some quick fixes

**Review by zomars — APPROVED** [review]
Next image isn't benefiting much since we request the image in full quality. Replaced with picture element that auto chooses light/dark mode.

**Pull request merged**